zha/application/gateway.py

```diff
@@ -314,7 +310,7 @@ def load_devices(self) -> None:
                 delta_msg,
                 zha_device.consider_unavailable_time,
             )
-        self.create_platform_entities()
+            self._maybe_create_device_entities(zha_device)
```

Comment:
should we drop "device" from the name here? this is going to create all entities and we have the concept of "device specific" entities w/ the counters and such. I think the name could be a bit confusing.